src/sandstorm/run-bundle.c++

```diff
@@ -2530,7 +2537,9 @@ private:
         altPortServer = altPortServer.attach(kj::mv(altPortService));
         for (auto port: config.ports.slice(1, config.ports.size())) {
           auto listener = fdBundle.consume(port, *io.lowLevelProvider);
-          auto promise = altPortServer->listenHttp(*listener);
+          auto promise = isHttpsPort(port)
+            ? tlsManager.listenHttps(*listener)
+            : altPortServer->listenHttp(*listener);
```

Comment:
Could the issue be here? If I can parse this, it means the altPortServer is only being reached if it's an HTTP port?